pom.xml

```diff
@@ -66,6 +67,11 @@
             <groupId>org.hibernate.validator</groupId>
             <artifactId>hibernate-validator</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springdoc</groupId>
+            <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+            <version>${open-api-webmvc-ui.version}</version>
+        </dependency>
```

Comment:
This PR contains changes from 2 tasks, please create PR with relevant changes only